packages/mui-codemod/src/deprecations/button-classes/button-classes.test.js

```diff
@@ -0,0 +1,78 @@
+import path from 'path';
+import { expect } from 'chai';
+import postcss from 'postcss';
+import { jscodeshift } from '../../../testUtils';
+import jsTransform from './button-classes';
+import { plugin as postcssPlugin } from './postcss-plugin';
+import readFile from '../../util/readFile';
+
+function read(fileName) {
+  return readFile(path.join(__dirname, fileName));
+}
+
+const postcssProcessor = postcss([postcssPlugin]);
+
+describe('@mui/codemod', () => {
+  describe('deprecations', () => {
+    describe('button-classes', () => {
+      describe('js-transform', () => {
+        it('transforms props as needed', () => {
+          const actual = jsTransform(
+            { source: read('./test-cases/actual.js') },
+            { jscodeshift },
+            { printOptions: { quote: 'double', trailingComma: true } },
```

Comment:
I wonder why your expected files are using double quotes 🤔

For me, prettier formats them to single quotes, so the codemods tests I've created have `quote: 'single'`

Reply:
Yeah i wondered same, my codemod was formatting them into double quotes, so changed `quote` to `double` in test.js. since this is not an important issue i hadn't spent much time investingating it. What do you think, does this work or shall spend some time to figure out?